microsoft-edge/webview2/releasenotes.md

```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
+The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a six week cadence.  Review the following content for up-to-date information about product announcements, additions and modifications to the API surface, and breaking changes.  
 
 > [!NOTE]
 > Re-compile your app after updating the NuGet package.  
 
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `pre-release package`.  
 
+
+## 0.9.628-prerelease  
+Release Date: Sept 09, 2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+   > [!IMPORTANT]
+    > **Announcement**: This prerelease will continue to release out of Edge 87 Branch. In the future, pre-release SDK releases will match the Edge Canary builds, whereas our normal releases will be in sync with Edge Stable.
+
+#### General
+*   > [!IMPORTANT]
+    > **Announcement**:  Visual Hosting is now in preview.
+* Added the CompositionController as an experimental interface. This lets your app connect WebView2 to Dcomp visual or WinComp ContainerVisual and provides API that allows the app to forward mouse and touch inputs to the WebView.
+
+#### .NET  
+
+*   .NET Binaries are now [Strongly Named]().  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).
+*   Updated NuGet Target to include WebViewLoader.dll.  \([\#228][GithubMicrosoftedgeWebviewfeedbackIssue228]\) & \([\#183][GithubMicrosoftedgeWebviewfeedbackIssue183]\).
+*   Updated WebResourceRequested to expose HttpRequestMessage and HttpResponseMessage APIs in .NET.  \([\#131][GithubMicrosoftedgeWebviewfeedbackIssue131]\).
+
+
+## 0.9.622  
+Release Date: Sept 09, 2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**: This SDK is the Release Candidate for WebView2 Win32 C/C++ GA. The GA version will likely have the same API interface and functionality.
+
+
+*   Disconnected [browser policies](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-policies). 
+*   Added [AllowSingleSignOnUsingOSPrimaryAccount][ReferenceWin3209538Icorewebview2experimentaloptionsGetIssinglesignonusingosprimaryaccountenabled] property on WebView2 environment options to enable conditional access for WebView.
+*   Updated ICoreWebView2NewWindowRequestedEventArgs to include WindowFeatures property, and its associated ICoreWebView2WindowFeatures.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).
+*   Updated System.Windows.Rect. to use System.Drawing.Rectangle instead of System.Windows.Rect \([\#235][GithubMicrosoftedgeWebviewfeedbackIssue235]\).
+*   Updated the Windows.open() function to handle "no parameter" case.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).
```

Comment:
"Updated NewWindowRequested event to handle window.open() call without parameters"?